test/borough/borough.repository.mock.ts

```diff
@@ -2,12 +2,17 @@ import {
   findManyRepoSchema,
   checkByIdRepoSchema,
   findCommunityDistrictsByBoroughIdRepoSchema,
+  findCapitalProjectsByBoroughIdCommunityDistrictIdRepoSchema,
 } from "src/borough/borough.repository.schema";
+// import { checkByCommunityDistrictIdRepoSchema, checkCommunityDistrictByIdMocks } from "src/community-district/community-district.repository.schema";
 import { generateMock } from "@anatine/zod-mock";
+import { CommunityDistrictRepositoryMock } from "test/community-district/community-district.repository.mock";
 
 export class BoroughRepositoryMock {
   numberOfMocks = 1;
 
+  communityDistrictRepoMock = new CommunityDistrictRepositoryMock();
```

Comment:
I would prefer we leveraged dependency injection instead of instantiating a new CommunityDistrictRepositoryMock object inside this mock.

It will also help with test consistency. The object instantiated here is different than the object instantiated directly in the test suite. It's only because we're using seeding that we get the same values. If we ever move away from seeding, we'll actually be getting different values

I made an example on a review branch: [247/ty/review](https://github.com/NYCPlanning/ae-zoning-api/commit/62c84a9ebef95cb93f6d4cf9882363c91c9fbea3)

Reply:
Updated to inject the whole object to make a bit more explicit where `checkCommunityDistrictByIdMocks` is coming from.